src/tester.rs

```diff
@@ -658,8 +664,26 @@ fn run_cmd(
         .spawn()
         .unwrap_or_else(|_| fatal(&format!("Couldn't run command {:?}.", cmd)));
 
-    let mut stderr = FileDescriptor::dup(child.stderr.as_ref().unwrap()).unwrap();
     let mut stdout = FileDescriptor::dup(child.stdout.as_ref().unwrap()).unwrap();
+    let mut stderr = FileDescriptor::dup(child.stderr.as_ref().unwrap()).unwrap();
```

Comment:
Why has this line moved? The rest of the code is, fairly consistently, `stderr` then `stdout`, so let's stick with that.

Reply:
Not sure how this happened I will switch these.